frontend/lib/norent/routes.ts

```diff
@@ -17,6 +17,9 @@ function createLocalizedRouteInfo(prefix: string) {
     /** The "About Your Letter" page explaining the letter building process in more detail. */
     aboutLetter: `${prefix}/the-letter`,
 
+    /** The Log-out page */
+    logOut: `${prefix}/log-out`,
+
```

Comment:
Actually do you mind removing the hyphen here and just calling it `/logout` so that it has parity with the tenant platform, for consistency's sake?  And naming the route `logout` instead of `logOut` too?